src/components/v5/common/CompletedAction/partials/CompletedExpenditureContent/CompletedExpenditureContent.tsx

```diff
@@ -88,8 +88,9 @@ const CompletedExpenditureContent: FC<CompletedExpenditureContentProps> = ({
   const tokensCount = getTokensNumber(expenditure);
   const stagedPaymentTokenSymbol =
     tokensCount === 1 &&
-    allTokens.find(({ token }) => token.tokenAddress === stages[0].tokenAddress)
-      ?.token.symbol;
+    allTokens.find(
+      ({ token }) => token.tokenAddress === stages[0]?.tokenAddress,
+    )?.token.symbol;
```

Comment:
Irrelevant fix but it's to prevent the Completed Action component from crashing. There's an isolated PR that also fixes this: https://github.com/JoinColony/colonyCDapp/pull/3627